dom-querying/main.js

```diff
@@ -0,0 +1,19 @@
+console.log('hello, world');
+
+let h1 = document.querySelector('h1');
+console.log('h1:', h1);
+console.dir(h1);
+
+let explanation = document.querySelector('#explanation');
+console.log('$explanation', explanation);
+console.dir(explanation);
+
+let hint = document.querySelector('.hint');
+console.log('$hint', hint);
+console.dir(hint);
+
+let parent = document.querySelectorAll('p');
+console.log('$paragraphs:', parent);
+
+let exampleLink = document.querySelectorAll('.example-link');
```

Comment:
These should all be `const`. We only ever want to use `let` if we plan on reassigning the variables to a different value. I recommend just always using `const` until you get an error, then switch to `let`

Reply:
Got it!